.pipelines/pipeline.yaml

```diff
@@ -314,31 +314,31 @@ stages:
       name: "cilium_e2e"
       displayName: Cilium
       pipelineBuildImage: "$(BUILD_IMAGE)"
-      testDropgz: ""
+      testDropgz: true
```

Comment:
Let's add separate stages instead of enabling it with current stages ? Testing a single thing at a time to make sure we know where the issue is.

cc: @tamilmani1989